renovate.json

```diff
@@ -1,4 +0,0 @@
-{
```

Comment:
Why is this being deleted?

Reply:
@onebytegone 
[In their docs](https://docs.renovatebot.com/config-overview/#onboarding-config), it's said that `renovate.json` has been deprecated, and for now it is only used when a `default.json` is not found. I couldn't test locally, but my assumption was that there could possibly be a bug in this fallback behavior. I've double checked their fallback implementation and it seems fine though, so this deletion was more of a test in this PR context.
Also, it is a cyclical reference to this same repository, which if used, should lead to `default.json` which is being kept, or, if somehow `default.json` is not used, it will fallback to itself (`renovate.json`) again and enter a loop.

But there is another possible fix I'd like to try. Should I revert this change for now?